Client/Frontend/Browser/Onboarding/Welcome/WelcomeViewCallout.swift

```diff
@@ -0,0 +1,478 @@
+    }
+    
+    private let primaryButton = RoundInterfaceButton(type: .custom).then {
+        $0.setTitleColor(.white, for: .normal)
+        $0.backgroundColor = .braveBlurple
+        $0.setTitle(" ", for: .normal)
+    }
+    
+    private let secondaryButtonContentView = UIStackView().then {
+        $0.axis = .horizontal
+        $0.spacing = 15.0
+        $0.isHidden = true
+        $0.alpha = 0.0
+        $0.layoutMargins = UIEdgeInsets(top: 0.0, left: 15.0, bottom: 0.0, right: 15.0)
+        $0.isLayoutMarginsRelativeArrangement = true
+    }
+    
+    private let secondaryLabel = UILabel().then {
+        $0.textColor = .bravePrimary
+        $0.textAlignment = .center
+        $0.numberOfLines = 0
+        $0.setContentHuggingPriority(.required, for: .horizontal)
+        $0.setContentCompressionResistancePriority(.required, for: .horizontal)
+        $0.setContentHuggingPriority(.required, for: .vertical)
+        $0.setContentCompressionResistancePriority(.required, for: .vertical)
+    }
+    
+    private let secondaryButton = RoundInterfaceButton(type: .custom).then {
+        $0.setTitleColor(.braveBlurple, for: .normal)
+        $0.backgroundColor = .clear
+        $0.setTitle(" ", for: .normal)
+        $0.setContentHuggingPriority(.required, for: .horizontal)
+        $0.setContentCompressionResistancePriority(.required, for: .horizontal)
+    }
+    
+    // MARK: - State
+    private(set) var state: WelcomeViewCalloutState?
+    
+    init(pointsUp: Bool) {
+        super.init(frame: .zero)
+        doLayout(pointsUp: pointsUp)
+        
+        [titleLabel, detailsLabel, primaryButton, secondaryButtonContentView].forEach {
+            contentView.addArrangedSubview($0)
+            
+            $0.alpha = 0.0
+            $0.isHidden = true
+        }
+        
+        [primaryButton, secondaryButton].forEach {
+            $0.contentMode = pointsUp ? .bottom : .top
+            $0.snp.makeConstraints {
+                $0.height.equalTo(44.0)
+            }
+        }
+        
+        let leftSpacer = UIView()
+        let rightSpacer = UIView()
+
+        [leftSpacer, secondaryLabel, secondaryButton, rightSpacer].forEach {
+            secondaryButtonContentView.addArrangedSubview($0)
+        }
+        
+        leftSpacer.snp.makeConstraints {
+            $0.width.equalTo(rightSpacer.snp.width)
+        }
+        
+        [titleLabel, detailsLabel].forEach {
+            $0.contentMode = pointsUp ? .bottom : .top
+            
+            $0.snp.makeConstraints {
+                $0.height.greaterThanOrEqualTo(17.0)
+            }
+        }
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        
+        layer.shadowColor = #colorLiteral(red: 0.4633028507, green: 0.4875121117, blue: 0.5066562891, alpha: 1).cgColor
```

Comment:
What colour is this? Should probably use a Brave design system colour if possible, otherwise use a hex initializer (`UIColor(rgb:)`)

Reply:
Used a hex initializer and added it struct UX https://github.com/brave/brave-ios/pull/4508/commits/19105ca671669ea1b1402edb5b3179cadd98e3cd